ui/guest/src/react/GuestProxy.tsx

```diff
@@ -40,7 +40,7 @@ import { GuestState } from '../store/models/GuestStore';
 import { notNullOrUndefined } from '../utils/object';
 import { forEach } from '../utils/array';
 import { popPiece, removeLastPiece } from '../utils/string';
-import { addAnimation } from '../utils/dom';
+import { addAnimation, getParentElementFromICEProps } from '../utils/dom';
```

Comment:
getParentElementFromICEProps => getElementByICEProps
move from dom to ElementRegistry

Reply:
can't rename to getElementByICEProps, because it does not retrieve the Element Itself, it uses the index on null, so that means it uses the parent dropZone

Reply:
also there is already a getElementByICEProps function

Reply:
moved.